src/main/kotlin/org/rust/lang/core/psi/ext/RsModificationTrackerOwner.kt

```diff
@@ -28,3 +31,13 @@ interface RsModificationTrackerOwner : RsItemElement {
      */
     fun incModificationCount(element: PsiElement): Boolean
 }
+
+fun findModificationTrackerOwnerFor(element: PsiElement): RsModificationTrackerOwner? {
```

Comment:
Maybe convert to extension function and rename to `findModificationTrackerOwner`?